Fundamentals/Setup/Install/installing-nightly-builds.md

```diff
@@ -0,0 +1,101 @@
+
+1. Adding the nightly feed as a NuGet source
+2. Finding the latest nightly version
+3. Installing the latest nightly version template
+
+But let's take things one step at a time.
+
+## Adding the nightly feed as a NuGet source
+
+The nuget feed containing the nightly builds are `https://www.myget.org/F/umbraconightly/api/v3/index.json`, you can either add this feed through the command line, or use your IDE of choice, in this guide we'll cover Visual Studio and Rider as well as using the command line.
+
+### Using the command line
+
+To add the nightly feed using the command line, open up your command prompt of choice and run this command:
+
+```
+dotnet nuget add source "https://www.myget.org/F/umbraconightly/api/v3/index.json" -n "Umbraco Nightly"
+```
+
+This will add the nightly feed as a source named `Umbraco Nightly`
+
+### Using Visual Studio
+
+To add the feed through Visual Studio, first open the NuGet settings by selecting the `Tools > NuGet Package Manager > Package Manager Settings` options from the top menu.
+
+![Open NuGet settings](images/VS/open-nuget-setttings.jpg)
+
+Now the options window will open, first select the `Package Sources` option, then click the `+` icon in the top bar, a new source will be added and automatically highlighted. Now enter your desired name for the feed in the `Name` input, and then enter the link `https://www.myget.org/F/umbraconightly/api/v3/index.json` into the source input and click `OK`.
+
+![Register the nightly feed](images/VS/registering-nightly-feed.jpg)
+
+Now the feed is added as a source named `Umbraco Nightly`
+
+### Using Rider
+
+To add the feed through Rider first click on the `NuGet` tab in the bottom, a menu blade will pop up, then choose the `Sources` tab. Now you must choose what NuGet configuration you wish to add the feed to, to add the feed globally add it to the `NuGet.Config` in `AppData\Roaming\NuGet`, by clicking that option. Lastly open up the `New Feed` dialog by clicking the green `+` button.
+
+![Open the new feed menu](images/Rider/open-add-feed.jpg)
+
+When you click the green `+` icon the new feed dialog will pop up, and the name and URL `https://www.myget.org/F/umbraconightly/api/v3/index.json`, to the fields, and press `OK`. Leave the User and Password empty and the `Enabled` box ticked.
+
+![Adding the feed](images/Rider/add-the-feed.jpg)
+
+Now the feed is added as a source named `Umbraco Nightly`
+
+## Finding the latest nightly version
+
+Now that the feed is added we're ready to install the nightly build, great! But this begs the question what is the version called we want to install? This is in particular an issue if we want to create a new site using the dotnet template, since the dotnet command does not allow us to use wildcards to install the newest version. Fortunately our tools can help us out here, we can see a list of available versions in both Visual Studio and Rider, and then use that version to install the template.
```

Comment:
```suggestion
Now that the feed is added we're ready to install the nightly build, great! However, this begs the question what is the version called we want to install? This is in particular an issue if we want to create a new site using the dotnet template, since the dotnet command does not allow us to use wildcards to install the newest version. Fortunately, our tools can help us out here, we can see a list of available versions in both Visual Studio and Rider, and then use that version to install the template.
```